{"name": "test_get_exercises", "status": "broken", "statusDetails": {"message": "TypeError: object Response can't be used in 'await' expression", "trace": "self = <test_async_exercises.TestExercises object at 0x10a8fc290>\nfunction_exercise = ExerciseFixture(request=CreateExerciseRequestSchema(title='Carry glass ground raise go scene try travel.', course_id='... build identify. Half know place. Yeah morning design book research son debate executive.', estimated_time='6 weeks')))\nexercises_client = <clients.exercises.exercises_client.ExercisesClient object at 0x10ac8aa80>\n\n    async def test_get_exercises(self,\n                               function_exercise: ExerciseFixture,\n                               exercises_client: ExercisesClient):\n        query = GetExercisesQuerySchema(courseId=function_exercise.response.exercise.course_id)\n>       response = await exercises_client.get_exercises_api(query=query)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       TypeError: object Response can't be used in 'await' expression\n\ntest_async_exercises.py:95: TypeError"}, "start": 1751656522570, "stop": 1751656522575, "uuid": "25d624c8-5de5-43fc-9326-a0949db4bb71", "historyId": "a9f3b85da6b3e5fb9f90acbdb9c35533", "testCaseId": "a9f3b85da6b3e5fb9f90acbdb9c35533", "fullName": "tests.exprcises.test_async_exercises.TestExercises#test_get_exercises", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.exprcises"}, {"name": "suite", "value": "test_async_exercises"}, {"name": "subSuite", "value": "TestExercises"}, {"name": "host", "value": "VIS-SSQ05N"}, {"name": "thread", "value": "77794-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.exprcises.test_async_exercises"}]}